src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelectorAll('li');
+
+function sortList(list) {
+  function strToNum(str) {
+    return +str.slice(1).split(',').join('');
+  }
+
+  return [...list].sort((item1, item2) =>
+    strToNum(item2.dataset.salary) - strToNum(item1.dataset.salary));
+}
+
+const sortedListOfEmployees = sortList(listOfEmployees);
+const ul = document.querySelector('ul');
+
+ul.innerHTML = '';
+
+for (const item of sortedListOfEmployees) {
+  ul.innerHTML += `
+    <li
+      data-position=${item.dataset.position}
+      data-salary=${item.dataset.salary}
+      data-age=${item.dataset.age}
+    >
+      ${item.textContent.trim()}
+    </li>
+  `;
```

Comment:
```suggestion
  ul.insertAdjacentHTML('beforeend', `
    <li
      data-position=${item.dataset.position}
      data-salary=${item.dataset.salary}
      data-age=${item.dataset.age}
    >
      ${item.textContent.trim()}
    </li>
  `);
```
Краще використати insertAdjacentHTML, бо innerHTML з кожним викликом перемальовує всі елементи, що були до того